aten/src/THC/generic/THCTensorTopK.cu

```diff
@@ -33,17 +33,17 @@ THC_API void THCTensor_(topk)(THCState* state,
   gatherTopK<real, INDEX_T, DIM, DIR>                                   \
     <<<grid, block, 0, THCState_getCurrentStream(state)>>>(             \
       inputInfo,                                                        \
-      sliceSize,                                                        \
-      k,                                                                \
-      inputSlices,                                                      \
+      static_cast<INDEX_T>(sliceSize),                                  \
```

Comment:
This needs a comment explaining why the `static_cast` is needed, so that someone doesn't "helpfully" remove it later.